src/course-tabs/CourseTabsNavigation.jsx

```diff
@@ -11,20 +11,21 @@ const CourseTabsNavigation = ({
 }) => (
   <div id="courseTabsNavigation" className={classNames('course-tabs-navigation', className)}>
     <div className="container-xl">
-      <Tabs
-        className="nav-underline-tabs"
-        aria-label={intl.formatMessage(messages.courseMaterial)}
-      >
-        {tabs.map(({ url, title, slug }) => (
-          <a
-            key={slug}
-            className={classNames('nav-item flex-shrink-0 nav-link', { active: slug === activeTabSlug })}
-            href={url}
-          >
-            {title}
-          </a>
-        ))}
-      </Tabs>
+      <nav className="nav flex-column nav-pills">
```

Comment:
what is the reason for using `div` inside `nav` instead of `ul/li`?

Reply:
using div is one of the options to achieve this but for readability ul/li is better to use.

Reply:
@ziafazal